app/forms/questionnaire_form.py

```diff
@@ -465,21 +465,23 @@ def get_answer_fields(
     routing_path_block_ids: dict[tuple, tuple[str, ...]] = {}
 
     if location and progress_store:
-        routing_path_block_ids = get_block_ids_for_calculated_summary_dependencies(
-            schema=schema,
-            location=location,
-            progress_store=progress_store,
-            path_finder=PathFinder(
-                schema=schema,
-                answer_store=answer_store,
-                list_store=list_store,
+        routing_path_block_ids = (
```

Comment:
This will store a map, so maybe name it `block_ids_by_section`. I don't necessary think it needs to be prefixed here with `routing_path` is there is enough context in the method.

Reply:
Updated 👍

Reply:
Sorry a bit confused, where is the routing_path prefix?